exporting/check_filters.c

```diff
@@ -4,7 +4,9 @@
 
 
 bool exporting_labels_filter_callback(const char *name, const char *value, RRDLABEL_SRC ls, void *data) {
-    (void)name;
+    if (name[0] == '_' )
+        return false;
```

Comment:
This filters `_*` from netdata.info. In Prometheus we apply this filter only for chart labels.

Reply:
Done with last commit, I also rebased this PR.